source/_integrations/modbus.markdown

```diff
@@ -219,6 +219,11 @@ modbus:
 ```
 
 {% configuration %}
+lazy_error_count:
+  description: Number of messages with error received before setting entity to unaviable. This parameter can be used to prevent spontanous errors to ruin statistic graphs.
```

Comment:
```suggestion
  description: Number of messages with error received before setting entity to unavailable. This parameter can be used to prevent spontaneous errors to ruin statistic graphs.
```